src/Uno.UI.Toolkit/Storage/StorageFile.wasm.cs

```diff
@@ -0,0 +1,15 @@
+#nullable enable
```

Comment:
Rename this file to `StorageFileHelper.wasm.cs` instead of `StorageFile.wasm.cs`